web/src/SidebarIcon.tsx

```diff
@@ -75,20 +76,32 @@ let SidebarIconRoot = styled.div`
     animation: ${glowDark} 2s linear infinite;
   }
   &.isNone {
-    background-color: ${ColorRGBA(Color.white, ColorAlpha.translucent)};
+    border-right: 1px solid ${Color.grayLighter};
+    box-sizing: border-box;
+    transition: border-color ${AnimDuration.default} linear;
+
+    svg {
+      fill: ${Color.grayLight};
+    }
   }
   .isSelected &.isNone {
-    background-color: ${ColorRGBA(Color.gray, ColorAlpha.translucent)};
+    border-right-color: ${Color.grayLightest};
+
+    svg {
+      fill: ${Color.grayLighter};
+    }
   }
 `
 
 export default class SidebarIcon extends PureComponent<SidebarIconProps> {
   render() {
     let icon = <span>&nbsp;</span>
     if (this.props.status === ResourceStatus.Warning) {
-      icon = <WarningSvg fill="#fff" height="9px" width="9px" />
+      icon = <WarningSvg fill={Color.white} width="10px" height="10px" />
```

Comment:
FYI: I removed the height and width when we're not adjusting the original size specified in the SVG file.